src/components/specialized/Wallets/Wallet/Register/index.js

```diff
@@ -12,12 +12,15 @@ class WalletRegister extends Component {
   }
 
   render () {
-    const { blockchain } = this.props
+    const { blockchain, pending } = this.props
+    const disabled = pending ? true : false
```

Comment:
Why this line ?
either you just do disabled = pending, or later you just use pending (basically it's same thing)